product_packaging_level_salable/tests/common.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.tests import Form, common, tagged
+
+TU_PRODUCT_QTY = 20
+PL_PRODUCT_QTY = TU_PRODUCT_QTY * 30
+
+
+@tagged("post_install", "-at_install")
+class Common(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
FYI, to avoid the `tracking_disable` context, you can use `BaseCommon` class
`from odoo.addons.base.tests.common import BaseCommon`